src/main/webapp/frame/framec.jsp

```diff
@@ -684,11 +690,11 @@
                             <field name="NUM">0</field>
                         </block>
                     </value>
-                    <value name="LENGTH">
+<%--                <value name="LENGTH">
```

Comment:
It really should just deleted - it looks like the block was made based on the wav player example published on learn.  In that example, each volume change was followed by a pause before another volume change was issued - and I think it was just assumed that the pause dictated how long the song should play - which is not really how it works.